app/services/katello/pulp/server.rb

```diff
@@ -0,0 +1,27 @@
+module Katello
+  module Pulp
+    class Server
+      def self.config(user_remote_id)
+        uri = URI.parse(SETTINGS[:katello][:pulp][:url])
+
+        Runcible::Instance.new(
+          :url => "#{uri.scheme}://#{uri.host.downcase}",
+          :api_path => uri.path,
+          :user => user_remote_id,
+          :timeout => SETTINGS[:katello][:rest_client_timeout],
+          :open_timeout => SETTINGS[:katello][:rest_client_timeout],
+          :oauth => {
+            :oauth_secret => SETTINGS[:katello][:pulp][:oauth_secret],
+            :oauth_key => SETTINGS[:katello][:pulp][:oauth_key]
+          },
+          :logging => {
+            :logger => ::Foreman::Logging.logger('katello/pulp_rest'),
+            :exception => true,
+            :info => true,
+            :debug => true
+          }
+        )
+      end
+    end
+  end
+end
```

Comment:
both https://github.com/Katello/katello/pull/5750 and https://github.com/Katello/katello/pull/5710 are also adding ways to fetch a runcible instance for a particular capsule, would we want to try to provide that here?